src/libstore/daemon.cc

```diff
@@ -222,7 +222,8 @@ struct ClientSettings
                     else if (!hasSuffix(s, "/") && trusted.count(s + "/"))
                         subs.push_back(s + "/");
                     else
-                        warn("ignoring untrusted substituter '%s'", s);
+                        warn("ignoring untrusted substituter '%s', you are not a trusted user.\n"
+                             "More information about 'trusted-substituters' option in nix.conf man page", s);
```

Comment:
```suggestion
                             "More information about 'trusted-substituters' option can be found in the nix.conf man page", s);
```

Reply:
I tried to keep the wording short, I'm not convinced your suggestion adds any value.
Is my current wording incorrect in English? I'm not a native speaker, I can't tell.

Reply:
In any case, you probably want to give instructions here, and those should be written in imperative.
```suggestion
                             "Run `man nix.conf` for more information on the `substituters` configuration option.", s);
```

@rapenne-s please add the detailed instructions to that section and link to it from the docs on the related options. (I don't really care where you put it, but `substituters` is the general concept, and the other things are more specific, so I suppose that is what people would stumble upon first when skimming around.)